data/account/participationRegistry.go

```diff
@@ -577,26 +627,46 @@ func scanRecords(rows *sql.Rows) ([]ParticipationRecord, error) {
 	results := make([]ParticipationRecord, 0)
 	for rows.Next() {
 		var record ParticipationRecord
-		var participationBlob []byte
-		var accountBlob []byte
+		var rawParticipation []byte
+		var rawAccount []byte
+		var rawVRF []byte
+		var rawVoting []byte
 		err := rows.Scan(
-			&participationBlob,
-			&accountBlob,
+			&rawParticipation,
+			&rawAccount,
 			&record.FirstValid,
 			&record.LastValid,
 			&record.KeyDilution,
+			&rawVRF,
 			&record.LastVote,
 			&record.LastBlockProposal,
 			&record.LastCompactCertificate,
 			&record.EffectiveFirst,
 			&record.EffectiveLast,
+			&rawVoting,
 		)
 		if err != nil {
 			return nil, err
 		}
 
-		copy(record.ParticipationID[:], participationBlob)
-		copy(record.Account[:], accountBlob)
+		copy(record.ParticipationID[:], rawParticipation)
+		copy(record.Account[:], rawAccount)
+
+		record.VRF = &crypto.VRFSecrets{}
```

Comment:
move this inside the if{} so that record.VRF is nil if there's no rawVRF?

Reply:
Fixed